pkg/cluster/install.go

```diff
@@ -33,7 +33,8 @@ func (m *manager) AdminUpdate(ctx context.Context) error {
 		steps.Condition(m.apiServersReady, 30*time.Minute),
 		steps.Action(m.ensureBillingRecord), // belt and braces
 		steps.Action(m.fixSSH),
-		steps.Action(m.fixPullSecret), // TODO(mj): Remove when operator deployed
+		steps.Action(m.fixPullSecret),     // TODO(mj): Remove when operator deployed
+		steps.Action(m.populateCreatedAt), // TODO(mikalai): Remove after a round of admin updates
```

Comment:
Created a work item for this so we don't forget: https://msazure.visualstudio.com/AzureRedHatOpenShift/_workitems/edit/9128557